miqi/agent/tools/request_action_confirmation.py

```
@@ -0,0 +1,116 @@
+"""request_action_confirmation — 危险动作最后确认工具（#646-v2，GPT 第五轮拍板）。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)